script/Arcueid/Day 5/05_00_ARC05_3_2B.txt

```diff
@@ -6,7 +6,7 @@
 [sha:b9ffb48a4d1a0dcdceeaf6c4a2b6a2d99cfdb51b]{
 -- Page 10007, Offset 9202.
 -- お腹減ってるでしょ、志貴？　さっきのおかわり、いる？　新しいの作ってあげよっか？」
-You must be hungry, right Shiki? Do you want seconds of what I made earlier? Or shall I make you something new?"
+You must be hungry, right Shiki? Do you want seconds of what I made earlier? Or shall I whip up something else?"
```

Comment:
"whip up something new" is a little much. If you want to get more colloquial 
"Want me to make you something else?" otherwise
"Or shall I make you something else."